package.json

```diff
@@ -10,8 +10,15 @@
     "test": "NODE_ENV=test jest --runInBand --detectOpenHandles --coverage",
     "kill-run": "npm run kill && npm run dev",
     "start": "NODE_ENV=production node ./server/index.js",
-    "heroku-postbuild": "NODE_ENV=production cd client && npm install && npm run build"
+    "heroku-postbuild": "NODE_ENV=production cd client && npm install && npm run build",
+    "lint-fix": "eslint server/ --fix",
+    "lint": "eslint server/",
+    "lint-react": "cd client && npm run lint"
```

Comment:
please add react lint fix to the script;
```json
"lint-react-fix": "cd client && npm run lint-fix"
```

Reply:
Ok